core/spec/models/spree/shipping_method_spec.rb

```diff
@@ -92,4 +95,31 @@ class DummyShippingCalculator < Spree::ShippingCalculator
       expect(shipping_method.deleted_at).not_to be_blank
     end
   end
+
+  describe '#available_to_display?' do
+    context 'when available on frontend' do
+      it { expect(frontend_shipping_method.available_to_display?(Spree::ShippingMethod::DISPLAY_ON_FRONT_END)).to be_truthy }
```

Comment:
@krzysiek1507 No this method will always return true | false. To be_truthy can also be used for the true check.
But still, for better understanding, I will change this ``to be_true``